ls1intum/Artemis
# [Feature] Enable markdown editor resizing for exercise create/edit components

### Checklist

### Motivation and Context

The markdown editor preview always kept the same size when creating or updating exercises.

### Description
### Steps for Testing

1. Start creating a new exercise (any type except quiz)
2. Write enough lines to extend past the default size of the preview
3. The preview should expand vertically instead of having a scrollbar

Example for how it looked before:
Editor|Preview
------|------
![grafik](https://user-images.githubusercontent.com/72132281/126475625-48c55483-f355-4942-9cac-2dc2ff7a0c50.png)|![grafik](https://user-images.githubusercontent.com/72132281/126475660-dc2feaee-3ce8-421f-b64f-5654fdc6bec3.png)

Example for how it looks now:
Editor|Preview
------|------
![grafik](https://user-images.githubusercontent.com/23650630/128741593-a8977fcb-57d6-442c-b379-9a717809596d.png)|![grafik](https://user-images.githubusercontent.com/23650630/128741622-14594d82-fb13-46ea-82c4-a5fb2f51a6f0.png)

You can play around with re-size icon
<img src="https://user-images.githubusercontent.com/23650630/128746121-ab076463-31ee-4a9d-97a1-1446e693019a.jpg" width="500"/>
